src/sensors.rs

```diff
@@ -6,7 +6,7 @@ use quick_error::quick_error;
 use redis::Commands;
 use redis::RedisError;
 
-use crate::api;
+use crate::api::sensors;
```

Comment:
It's already re-exported through `spaceapi_server::api::sensors`. Do you think a second re-export would help?

Reply:
I think I wanted that users don't need to distinguish between `spaceapi::sensors` and `spaceapi_server::sensors`. But this may be more confusing then helping.

Reply:
Ah `spaceapi_server::sensors` isn't even public.